deploy.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+VAULT_TOKEN=$1
+GIT_BRANCH=$2
+
+#need to get the environment from the branch name
+if [ "$GIT_BRANCH" == "develop" ]; then
+	ENVIRONMENT="dev"
+elif [ "$GIT_BRANCH" == "alpha" ]; then
+        ENVIRONMENT="alpha"
+elif [ "$GIT_BRANCH" == "staging" ]; then
+	ENVIRONMENT="staging"
+elif [ "$GIT_BRANCH" == "master" ]; then
+	ENVIRONMENT="prod"
+else
+	echo "Unknown Git branch $GIT_BRANCH"
+	#exit 1
+fi
+
+GOOGLE_PROJECT=broad-avram-$ENVIRONMENT
+SERVICE_VERSION=version1
```

Comment:
What is this used for? Does this need to be something other than a fixed value?

Reply:
I'm removing the version name for now (I just adopted it from Bond) but we'll probably want versioned APIs at some point in the future